resources/checkout-order-line-item-type.json

```diff
@@ -0,0 +1,67 @@
+{
+  "key": "checkout-order-line-item",
+  "name": {
+    "en": "Checkout order line item"
+  },
+  "description": {
+    "en": "Checkout order line item definition"
+  },
+  "resourceTypeIds": ["line-item", "custom-line-item"],
+  "fieldDefinitions": [
+    {
+      "name": "schedule",
+      "type": {
+        "name": "String"
+      },
+      "required": true,
```

Comment:
All of the custom type attributes should be defined as `required = false` since the custom type is used also by the checkout for non subscription orders. We just need to document which attributes are mandatory for subscription job to work.